shoop/core/api/orders.py

```diff
@@ -45,6 +56,34 @@ class Meta:
         model = Order
 
 
+def handle_payment_creation(request, order):
+    serializer = PaymentSerializer(data=request.data)
+    if serializer.is_valid():
+        data = serializer.validated_data
+        order.create_payment(
+            Money(data["amount_value"], data["currency"]),
+            data["payment_identifier"],
+            data.get("description", "")
+        )
+        return Response({'status': 'payment created'}, status=status.HTTP_201_CREATED)
+    else:
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
This function shouldn't be public. It's just a helper for OrderViewSet. Also move it under OrderViewSet for making the code easier to read (less jumping up/down in the file).